src/utils/pitchingStatsConfig.js

```diff
@@ -0,0 +1,24 @@
+import { getStatValue } from './getStatValue'
+import { calculateEarnedRunAverage } from './pitchingStatsCaclulations'
+
+const pitchingStatsConfig = [
```

Comment:
There's a more sleek, less repetitive way of creating this array, same for hitting:

```javascript
const pitchingStatsConfig = []
['GP', 'GS', 'W', ...].forEach(stat => {
   let callBackFunction = getStatValue;
   if (stat === 'ERA') {
      callBackFunction = calculateEarnedRunAverage
   } else if ( // other specific stats calcuations
   ...
   
   pitchingStatsConfig.push({ key: 'GP', callBack: callBackFunction})
})
```